### DISCUSSION TITLE
EDC vs IDS Connector

### DISCUSSION THREAD

**Discussion by dilipdhankecha** [International-Data-Spaces-Association/DataspaceConnector#132] [Q&A] [answered]
How are [EDC](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector) and [IDS connector](https://github.com/International-Data-Spaces-Association/DataspaceConnector) similar? 

What are the differences between them and what would you recommend is better of the two? 

We are currently thinking of integrating one of these 2 in our product offering, and wanted to understand more on this. Also, if you can help us in understanding how CatenaX, GaiaX, TractusX etc. are linked to EDC and IDS, that would be helpful in terms of streamlining our approach. 

Thanks in advance!

**Comment by sebk46** [comment, marked as answer]
Hi @dilipdhankecha,

You have already seen the answer in the [EDC repo](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/1037) - we can confirm this and we highly welcome the EDC initiative, however, we also would like to bring in the DSC perspective.

As you can see in the EDC repo, the Eclipse project receives strong contributions and support by different committers, but it depends on the status of you project which Connector you should use. The EDC is in a development stage with a first release planned for summer. The DSC can be used already today. The background why the decision was taken to develop EDC is already explained in the EDC discussion. Let me highlight some of the current benefits with DSC:

**Approved as IDS Connector:**
- Sets the standard for IDS Connectors and thus represents a reference implementation
- IDS ready certified and IDSA incubating
- Already in use in productive data-ecosystems and many Fraunhofer-projects and thus consolidated in its development and architecture (e.g. Mobility Data Space)
- Proven usage control features
- Findings from the development and productive operation of the DSC influence the IDS Reference Architecture Model.
- DSC is the primary IDS Connector of IDS testbed and thus IDS certification

**IDS Compatibility positioned:**
- Proven compatibility to other IDS infrastructure components besides DAPS (Broker, Clearing-House, App-Store …)
- A closed data-ecosystem can be built with the DSC (see feature DAPS whitelisting), if every ecosystem has its own DAPS

**Direct comparison with EDC:**
- Same IDS protocol support as EDC (IDS Multipart)
- EDC benefits from the experience gained in the DSC, DSC serves as a template for IDS-developments in EDC
- DSC provides a graphical user interface to administer and operate the DSC (Apache 2.0)
- DSC has no separation of control plane and data plane enhancing legal security and to not lose usage control over the shared data
- DSC has smaller Code base helping to understand the code quicker
